sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_visits_daily_v2/schema.yaml

```diff
@@ -0,0 +1,169 @@
+fields:
+- name: submission_date
+  type: DATE
+  mode: NULLABLE
+  description: Date of visit activity
+- name: client_id
+  type: STRING
+  mode: NULLABLE
+  description: Unique ID for the client installation.
+- name: newtab_visit_id
+  type: STRING
+  mode: NULLABLE
+  description: The id of newtab visit
+- name: app_version
+  type: INTEGER
+  mode: NULLABLE
+  description: Fx major version number
+- name: os
+  type: STRING
+  mode: NULLABLE
+- name: channel
+  type: STRING
+  mode: NULLABLE
+- name: locale
+  type: STRING
+  mode: NULLABLE
+- name: country
+  type: STRING
+  mode: NULLABLE
+- name: homepage_category
+  type: STRING
+  mode: NULLABLE
+  description: Setting for home page.
+- name: newtab_category
+  type: STRING
+  mode: NULLABLE
+  description: Setting of the newtab page
```

Comment:
These are both confusing so it'd be good to have more details on what they mean and what the possible values are